src/metrics/history.py

```
@@ -1,17 +1,170 @@
+                          '/elife-news/composing-reproducible-manuscripts-using-r-markdown': 'cad57bcf',
+                          '/elife-news/elife-labs-presenting-proteopedia-sharing-macromolecule-concepts-online': 'a9c66d96',
+                          '/elife-news/elife-labs-what-manuscripts': '0f2f45d5',
+                          '/elife-news/hack-cambridge-recurse-entries-explore-knowledge-direct-scichat': 'cb73dd25',
+                          '/elife-news/international-image-interoperability-framework-iiif-science-publishers': 'aabe94cd',
+                          '/elife-news/introducing-lens-writer': '417bcedc',
+                          '/elife-news/introducing-our-new-experiment-lens-writer': '417bcedc',
+                          '/elife-news/labs': '',
+                          '/elife-news/proteopedia-sharing-macromolecule-concepts-online': 'a9c66d96',
+                          '/elife-news/toward-publishing-reproducible-computation-binder': 'a7d53a88',
+                          '/elife-news/what-is-manuscripts': '0f2f45d5',
+                          '/elife-news/writing-scholarly-documents-manuscripts': '0f2f45d5'}}]},
+
+    'press-package': {
+        'frames': [
+            {'starts': GA4_SWITCH,
+             'ends': None,
+             'id': 3,
+             'prefix': '/for-the-press'},
+
+            {'starts': '2017-06-01',
+             'ends': GA4_SWITCH - one_day,
+             'id': 2,
+             'prefix': '/for-the-press'},
+
+            {'starts': None,
+             'ends': '2017-05-31',
+             'id': 1,
+             'path-map-file': 'press-package-path-map.txt',
+             'prefix': '/elife-news',
+             'redirect-prefix': '/for-the-press'}]}}
+
+# --- spec
+
 def date_wrangler(v):
     if isinstance(v, str):
         return datetime.strptime(v, "%Y-%m-%d").date()
+    if isinstance(v, datetime):
+        return v.date()
     return v
```

what is using this? raise valueerror rather than return what is given